angular-workspace/projects/ni/nimble-angular/src/directives/table-column/anchor/tests/nimble-table-column-anchor-router-link-with-href.directive.spec.ts

```diff
@@ -3,11 +3,14 @@ import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testin
 import { Router } from '@angular/router';
 import { CommonModule, Location } from '@angular/common';
 import { RouterTestingModule } from '@angular/router/testing';
+// Must import here. Cannot re-export from directive, because cannot import in non-test code.
+// eslint-disable-next-line no-restricted-imports
```

Comment:
Similar to below, express this as a workaround until nimble-angular exposes `testing` entrypoints and link to the same comment